syntax = "proto3";

service Sender {
    rpc SendMessage (SendRequest) returns (SendResponse);
}

message SendRequest {
    string message = 1;
}

message SendResponse {
    bool is_success = 1;
}